transport_nantes/topicblog/views.py

```diff
@@ -1084,3 +1084,44 @@ def send_received_handler(sender, mail_obj, send_obj, *args, **kwargs):
         except Exception as e:
             logger.error(
                 f"Error while updating send_record : {e}")
+
+
+@receiver(open_received)
+def open_received_handler(sender, mail_obj, send_obj, *args, **kwargs):
+    """Handle AWS SES open_received notifications
+
+    AWS Receiver
+    This function will run when a open_received is received from
+    Amazon SES.
+    The signal is sent from django_ses' view.
+
+    We voluntarily ignore open_received notifications because
+    we already have the beacon view handling that for us.
```

Comment:
Good.  Maybe just add that (here and for click_recieved) that this signal introduces AWS tracking that we don't want because it triggers warnings from ublock origin and obfuscates links, and so whilst we catch and log it, we in principle have turned it off on the AWS console.

Reply:
https://github.com/transport-nantes/tn_web/blob/99795780de25a22910b1bf64edffb466f7f209ca/transport_nantes/topicblog/views.py#L1118

I already do that :) Only the click triggers the uBlock warning

Reply:
Added the quoted sentence to both docstrings